app/views/results_submission/_results_submission_panel.html.erb

```diff
@@ -21,8 +21,8 @@
       <p>Make sure the schedule on the WCA website actually reflects what happened during the competition.</p>
       <p>
         Please also make sure to include any other additional details required by the
-        <%= link_to "'Submitting competition results' section of the delegate crash course",
-          panel_delegate_crash_course_path %>.
+        <%= link_to "'Competition Results' section of the Delegate Handbook",
+          "https://documents.worldcubeassociation.org/edudoc/delegate-handbook/delegate-handbook.pdf#competition-results" %>.
```

Comment:
1. I tried opening this link, but it's not scrolling to the section in the handbook. Is this something that is supported in at least some browsers?
2. Having the same URL hardcoded in two places looks bit odd, but I cannot think of a way to solve it especially because there is a hash parameter. Just mentioning here so that you or somebody else will be able to find a solution.

Reply:
1. Yes; on Firefox the link takes me straight to the section. It should be supported in other browses as AFAIK it's the same as heading tag hashes in URLs, but if it doesn't work in some browser configuration that's nbd to me since otherwise the link works and the section is explicitly called out here.
2. I'm not sure how you'd prefer to handle a constant like this across Rails code and React code, open to suggestions if it's a priority fix. Changing this around would also be easy in a subsequent commit if required.